### PR TITLE
Added markup and style to edit event page.

### PR DESCRIPTION
**Pull request by marioabreu**
The dropdown input need to be refactored.

### CODE REVIEW

File: app/assets/stylesheets/components/_form.sass [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+$Form-input-border: 2px
+$Form-input-width-hashtag: 212px
+$Form-spacing-base: $Theme-spacing-base
+$Form-spacing-small: $Theme-spacing-small
+$Form-spacing-tiny: $Theme-spacing-small / 2
+$Form-width: 425px
+
+.Form
+  margin: 0 auto
+  width: $Form-width
```
Comment (azevedo) [inline app/assets/stylesheets/components/_form.sass:10]: Why are you setting a fixed width?

File: app/assets/stylesheets/components/_form.sass [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+$Form-fontSize-input: Theme-fontSize(text, base, font-size)
+$Form-height: 40px
+$Form-input-border: 2px
+$Form-input-width-hashtag: 212px
+$Form-spacing-base: $Theme-spacing-base
+$Form-spacing-small: $Theme-spacing-small
+$Form-spacing-tiny: $Theme-spacing-small / 2
+$Form-width: 425px
+
+.Form
```
Comment (azevedo) [inline app/assets/stylesheets/components/_form.sass:7]: This whole component complete violates the separation between skin and structure. It basically does everything and works for the particular page you had to implement.
I hope you're aware of this

File: app/assets/stylesheets/components/_form.sass [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+.Form-label
+  color: $Form-color-label
+  display: block
+  padding: $Form-spacing-base 0 $Form-spacing-tiny 0
+
+.Form-input
+  border-radius: $Form-input-border
+  border: 2px solid $Form-border-input
+  font-color: $Form-color-filled
```
Comment (azevedo) [inline app/assets/stylesheets/components/_form.sass]: This property is not valid CSS.
I've told you many times: Don't make CSS previous to see how it renders in the browser and you feel the actual need to implement things in CSS.
This proves you do things without actually paying attention to what each individual property does.
You're learning, so you need to pay extra attention to these details.

File: app/assets/stylesheets/components/_form.sass (hunk 2 above)
Comment (azevedo) [inline app/assets/stylesheets/components/_form.sass:7]: Basically each input should be its own component where you would set their style. This class has too many responsibilities and makes it hard to reuse

File: app/assets/stylesheets/components/_theme.scss [hunk 4]
```diff
@@ -121,3 +121,17 @@ html{
   height: 100%;
   text-rendering: optimizeLegibility;
 }
+
+select {
```
Comment (azevedo) [inline app/assets/stylesheets/components/_theme.scss:125]: How come a reset does so much and even sets such specific styles such as width and height

File: app/views/events/_fields.slim [hunk 5]
```diff
@@ -1,7 +1,21 @@
...
-  = f.text_field :hashtag
-  = f.text_field :interest_list, value: @event.interest_list.join(',')
-  = f.submit
+.Form
+  .Form-label
+    = f.label :name_name
+  = f.text_field :name, class: 'Form-input', required: true
+  .Form-label
+    = f.label :event_hashtag
+  = f.text_field :hashtag, class: 'Form-input Form--hashtag'
```
Comment (azevedo) [inline app/views/events/_fields.slim]: Try to use modifiers only on the top component

File: app/assets/stylesheets/components/_form.sass (hunk 2 above)
Comment (marioabreu) [inline app/assets/stylesheets/components/_form.sass:7]: I will refactor this.

File: app/assets/stylesheets/components/_theme.scss (hunk 4 above)
Comment (marioabreu) [inline app/assets/stylesheets/components/_theme.scss:125]: I had some trouble styling the dropdown input.
I will try another way.

File: app/views/events/_fields.slim (hunk 5 above)
Comment (marioabreu) [inline app/views/events/_fields.slim]: Modifiers are rarely used. When we are using a modifier, in some cases, can this be a smell to refactor the components in small components?

File: app/assets/stylesheets/components/_form.sass (hunk 1 above)
Comment (marioabreu) [inline app/assets/stylesheets/components/_form.sass:10]: I refactored this but i'm still setting the width to 425px.

File: app/assets/stylesheets/components/_form.sass (hunk 3 above)
Comment (marioabreu) [inline app/assets/stylesheets/components/_form.sass]: You are right.
I will try harder.

**Pull request merged**